mkstage4.sh

```diff
@@ -79,20 +78,23 @@ shift;OPTIONS="$@"
 
 # Excludes:
 EXCLUDES="\
+--exclude=.bash_history \
+--exclude=dev/* \
+--exclude=lost+found \
+--exclude=media/* \
+--exclude=mnt/*/* \
 --exclude=proc/* \
+--exclude=run/* \
 --exclude=sys/* \
 --exclude=tmp/* \
---exclude=mnt/*/* \
+--exclude=usr/portage/* \
 --exclude=var/lock/* \
 --exclude=var/log/* \
---exclude=var/run/* \
---exclude=.bash_history \
---exclude=lost+found \
---exclude=usr/portage/*"
+--exclude=var/run/*"
 
 if [ "$TARGET" == "/" ]
 then
-  EXCLUDES+=" --exclude=$STAGE4_FILENAME"
+  EXCLUDES+=" --exclude=${STAGE4_FILENAME#/}"
```

Comment:
It's not a shebang: http://www.tldp.org/LDP/abs/html/parameter-substitution.html#PSOREX2